apps/react-18-tests-v8/just.config.ts

```diff
@@ -1,3 +1,19 @@
-import { preset } from '@fluentui/scripts-tasks';
+import { preset, task, typeCheckWithConfigOverride } from '@fluentui/scripts-tasks';
 
 preset();
+
+task('type-check', () =>
+  typeCheckWithConfigOverride(config => {
+    return {
```

Comment:
we need to override path resolution to force react and global type acquisition to the application scope. without this TS wont understand `resolutions` and would merge monorepo types (react v17 with react 18 which will explode on type check )